### ISSUE TITLE
Does not work along with company-complete

### ISSUE THREAD

**Issue by avatar-lavventura** [expez/company-quickhelp#113]
I was not able to make it work, what may be the reason? `C-c h` is not triggered as well. 

my config file:

```
(company-quickhelp-mode)

(eval-after-load 'company
  '(define-key company-active-map (kbd "C-c h") #'company-quickhelp-manual-begin))

(setq company-auto-complete t)

(eval-after-load 'company
  '(progn
     (define-key company-active-map (kbd "TAB") 'company-complete-common-or-cycle)
     (define-key company-active-map (kbd "<tab>") 'company-complete-common-or-cycle)))

(setq company-frontends
      '(company-pseudo-tooltip-unless-just-one-frontend
        company-preview-frontend
        company-echo-metadata-frontend))

(setq company-auto-complete t)
(global-set-key (kbd "C-c C-k") 'company-complete)

```

**Comment by dgutov** [comment]
Does `company-show-doc-buffer` work? It's normally bound to `C-h` while the completion is active.

If it does not, the current backend does not support to documentation display. You can find out the currently used backend with `M-x company-diag`.

**Comment by avatar-lavventura** [comment]
`M-x company-show-doc-buffer` returns:

```
abs(n: SupportsAbs[_T], /) -> _T

Return the absolute value of the argument.
```

------

C-h is bind to delete a character on my end; while completion is active, `C-h`  deletes a character.

Output of `M-x company-diag`; my current backend: https://gist.github.com/avatar-lavventura/c37df6f9f35e377c89d880f9be9ec664

**Comment by dgutov** [comment]
OK.

Please clarify: does it work with other backends? In `.el` files, for instance.

**Comment by avatar-lavventura** [comment]
I am not sure it does not do work do anything, maybe I was not able to make proper init.el file.

Is there any `.emacs` configurations that I can try, which works on your end?

**Comment by dgutov** [comment]
Here's the problem: you're resetting the value of `company-frontends` after enabling `company-quickhelp-mode`. Thereby undoing the effect of that function.

Do it in the reverse: move `(company-quickhelp-mode)` to the last line.

**Comment by avatar-lavventura** [comment]
ah I tried it but I does not work :-(

---------

updated `.emacs:`

```emacs
(require 'package)
(setq package-archives '(("gnu" . "https://elpa.gnu.org/packages/")
                         ("melpa" . "https://melpa.org/packages/")
                         ("org" . "https://orgmode.org/elpa/")))

(add-to-list 'package-archives
             '("MELPA Stable" . "http://stable.melpa.org/packages/") t)

(package-initialize)

(when (not package-archive-contents)
  (package-refresh-contents))

(if (not (package-installed-p 'company-quickhelp))
    (progn
      (package-install 'company-quickhelp)))

(add-hook 'after-init-hook 'global-company-mode)
(setq company-auto-complete t)

(setq company-auto-complete t)
(global-set-key (kbd "C-c C-k") 'company-complete)

(company-quickhelp-mode)  ;; moved to the end of the file

(eval-after-load 'company
  '(define-key company-active-map (kbd "C-c h") #'company-quickhelp-manual-begin))
```

**Comment by dgutov** [comment]
> ah I tried it but I does not work :-(

Does it work in `emacs-lisp-mode`? Or in `*scratch*`?

**Comment by avatar-lavventura** [comment]
Like I only used text code on my previoes comment in `.emacs` file and deleted all the other stuff but there was no affect.

**Comment by dgutov** [comment]
Could you rephrase that? I'm not sure I understand you well.

**Comment by avatar-lavventura** [comment]
Sorry I am just unable to make it work.

**Comment by dgutov** [comment]
It doesn't work in `*scratch*` either?

Are you using a graphical Emacs? Not in terminal?
